src/pages/discover.js

```diff
@@ -60,7 +108,7 @@ function useCollectives(query) {
 
 const DiscoverPage = ({ router }) => {
   const { query } = router;
-  const { collectives, offset, total, show, sort, tags = [] } = useCollectives(query);
+  const { collectives, offset, total, show, tags, collective = [] } = useCollectives(query);
```

Comment:
As we're moving to GraphQL we don't need `useCollectives` anymore. Instead Apollo will inject the data in `data` prop.